definitions/pipelines/rnaseq_star.cwl

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "RNA-Seq alignment and transcript/gene abundance workflow"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: InlineJavascriptRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outSAMattrRGline:
+        type:
+            type: array
+            items:
+                type: array
+                items: string
+    genomeDir:
+        type: Directory
+    sjdbGTFfile:
+        type: File
+    sample_name:
+        type: string
+    runMode:
+        type: string
```

Comment:
Is this ever going to be anything other than `alignReads` for this workflow?  If not then we should drop this at the top level and make the inputs as simple as we can.

Reply:
made it default